tests/support_server/wait.pm

```diff
@@ -21,25 +20,9 @@ use lockapi;
 use mmapi;
 
 sub run {
-    $testapi::username = "root";
-    $testapi::password = "linux";
 
+    my $self = shift;
 
-    assert_screen( "autoyast-system-login-console", 300 );
-    
-    type_string "$username\n";
-    sleep 5;
-#        assert_screen "password-prompt", 10;
-    type_password;
-    type_string "\n";
-    sleep 5;
-    
-    
-    script_output("
-        curl -f -v " . autoinst_url . "/data/supportserver/proxy.conf | sed -e 's|#AUTOINST_URL#|" . autoinst_url . "|g' >/etc/apache2/vhosts.d/proxy.conf
-        systemctl restart apache2.service
-    ");
-    
```

Comment:
there is also significat time before the dependent machine start, but ok, maybe wait idle is safer. Will check if it does not delay support server setup too much